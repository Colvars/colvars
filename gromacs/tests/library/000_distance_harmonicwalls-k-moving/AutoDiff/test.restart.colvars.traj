# step         one                   fa_one                 E_harmonicwalls1      W_harmonicwalls1     
          20    3.21706507941027e-01 -5.04936011690103e-02   7.61712404103452e-03 -6.82181597116614e-02
          21    3.21488025965126e-01 -5.04570360255235e-02   7.60609609369317e-03 -6.82181597116614e-02
          22    3.21268092808886e-01 -5.04202280124951e-02   7.59500296615677e-03 -6.82181597116614e-02
          23    3.21057495786911e-01 -5.03849824948974e-02   7.58438832759059e-03 -6.82181597116614e-02
          24    3.20865718609295e-01 -5.03528866664516e-02   7.57472871547719e-03 -6.82181597116614e-02
          25    3.20700464945098e-01 -5.03252298132116e-02   7.56641000165081e-03 -6.82181597116614e-02
          26    3.20567020111947e-01 -5.03028964859354e-02   7.55969584988865e-03 -6.82181597116614e-02
          27    3.20468795696520e-01 -5.02864576477696e-02   7.55475568463470e-03 -6.82181597116614e-02
          28    3.20407042948645e-01 -5.02761227078852e-02   7.55165067679950e-03 -6.82181597116614e-02
          29    3.20380905725098e-01 -5.02717483821524e-02   7.55033665570759e-03 -6.82181597116614e-02
          30    3.20388186953108e-01 -5.02729669684721e-02   7.55070270020640e-03 -6.82181597116614e-02
          31    3.20425433699984e-01 -5.02792005840294e-02   7.55257532077276e-03 -6.82181597116614e-02
          32    3.20488306089924e-01 -5.02897229072097e-02   7.55573682505954e-03 -6.82181597116614e-02
          33    3.20572027134761e-01 -5.03037344612736e-02   7.55994771973687e-03 -6.82181597116614e-02
          34    3.20671808439759e-01 -5.03204338604780e-02   7.56496792515161e-03 -6.82181597116614e-02
          35    3.20783047406248e-01 -5.03390508139096e-02   7.57056655367284e-03 -6.82181597116614e-02
          36    3.20901965872313e-01 -5.03589530083903e-02   7.57655397974805e-03 -6.82181597116614e-02
          37    3.21025316819200e-01 -5.03795970228613e-02   7.58276707751523e-03 -6.82181597116614e-02
          38    3.21150782173627e-01 -5.04005949045782e-02   7.58908928876491e-03 -6.82181597116614e-02
          39    3.21277002920381e-01 -5.04217192087549e-02   7.59545222265334e-03 -6.82181597116614e-02
          40    3.21403441318619e-01 -5.04428799390841e-02   7.60182880183093e-03 -6.82181597116614e-02
