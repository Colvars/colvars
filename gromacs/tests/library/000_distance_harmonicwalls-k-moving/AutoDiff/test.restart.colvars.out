colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: Summary of compile-time features available in this build:
colvars:   - C++ language version: 201703
colvars:   - Multiple replicas: not available
colvars:   - Lepton custom functions: available
colvars:   - Tcl interpreter: not available
colvars: ----------------------------------------------------------------------
colvars: Start colvars Initialization.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration:
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars:   prot_N (10 atoms)
colvars:   prot_CA (10 atoms)
colvars:   prot_C (10 atoms)
colvars:   prot_O (10 atoms)
colvars: # smp = "cvcs" [default]
colvars: SMP parallelism will be applied to Colvars components.
colvars:   - SMP parallelism: enabled (num. threads = 2)
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # defaultInputStateFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "one"
colvars:     Initializing a new "distance" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group1"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 4 atoms requested: total mass = 54.028, total charge = -0.72.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group2"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.05
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 0.05 [default]
colvars:   # hardLowerBoundary = on [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   # targetLowerWalls =  [default]
colvars:   # targetUpperWalls =  [default]
colvars:   Initializing a new "harmonicwalls" instance.
colvars:   # name = "harmonicwalls1" [default]
colvars:   # colvars = { one }
colvars:   # stepZeroData = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # outputEnergy = on
colvars:   # outputFreq = 10 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # forceConstant = 0.004184
colvars:   # decoupling = off [default]
colvars:   # targetForceConstant = 0.0004184
colvars:   Initial step for restraint change: 0
colvars:   # targetNumSteps = 10
colvars:   # targetEquilSteps = 0 [default]
colvars:   # targetNumStages = 0 [default]
colvars:   # outputAccumulatedWork = on
colvars:   # lambdaSchedule =  [default]
colvars:   # lambdaExponent = 1 [default]
colvars:   # lowerWalls = { 0.01 }
colvars:   # upperWalls = { 0.02 }
colvars:   # lowerWallConstant = 0.004184 [default]
colvars:   # upperWallConstant = 0.004184 [default]
colvars:   # targetLowerWalls = { 0 } [default]
colvars:   Target lower walls were not provided.
colvars:   # targetUpperWalls = { 0 } [default]
colvars:   Target upper walls were not provided.
colvars:   The lower wall force constant for colvar "one" will be rescaled to 1.6736 according to the specified width (0.05).
colvars:   The upper wall force constant for colvar "one" will be rescaled to 1.6736 according to the specified width (0.05).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Current simulation parameters: initial step = 0, integration timestep = 1
colvars: Updating atomic parameters (masses, charges, etc).
colvars: Re-initialized atom group for variable "one":0/0. 4 atoms: total mass = 54.028, total charge = -0.72.
colvars: Re-initialized atom group for variable "one":0/1. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - GROMACS engine:
colvars:   Abraham2015 https://doi.org/10.1016/j.softx.2015.06.001
colvars: 
colvars: - Colvars module:
colvars: - Optimal rotation via flexible fitting:
colvars: - distance colvar component:
colvars: - harmonicWalls colvar bias implementation:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - Colvars-GROMACS interface:
colvars:   Fiorin2024 https://doi.org/10.1021/acs.jpcb.4c05604
colvars: 
colvars: The final output state file will be "test.restart.part0002.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: Loading state from unformatted memory.
colvars:   Restarting collective variable "one" from value: 0.321707
colvars:   Restarted harmonicwalls bias "harmonicwalls1" with step number 20.
colvars: ----------------------------------------------------------------------
colvars: ----------------------------------------------------------------------
colvars: End colvars Initialization.
colvars: 
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.part0002.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.part0002.colvars.traj".
colvars: Saving collective variables state to "test.restart.part0002.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.part0002.colvars.traj".
colvars: Saving collective variables state to "test.restart.part0002.colvars.state".
colvars: Saving collective variables state to "test.restart.part0002.colvars.state".
